src/core/tests/frontend/paddle/test_models/gen_scripts/generate_where_index.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# where paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import sys
+import paddle
+from paddle.fluid.layer_helper import LayerHelper
```

Comment:
Can you use paddle 2.1 api instead of fluid which is obsolete after paddle 1.8 to build unit tests?

Reply:
down.
But there is a little diference between `paddle.nonzero` and `where_index`.
With the parameter `as_tuple=True`, it will convert the output to tuple.